kivy/uix/scrollview.py

```diff
@@ -763,8 +763,8 @@ def on_scroll_move(self, touch):
         if ud['mode'] == 'unknown':
             ud['dx'] += abs(touch.dx)
             ud['dy'] += abs(touch.dy)
-            if ((ud['dx'] > self.scroll_distance) or
-                    (ud['dy'] > self.scroll_distance)):
+            if ((ud['dx'] > self.scroll_distance and self.do_scroll_x) or
+                    (ud['dy'] > self.scroll_distance and self.do_scroll_y)):
```

Comment:
You are correct, though that check is superfluous - one is using scrollview with deliberately disabled both axis - no scrolling. For now I moved it.